docker-compose.yml

```diff
@@ -28,15 +28,43 @@ services:
       - ./api/test:/opt/landgriffon-api/test
     env_file: .env
     environment:
+      - NODE_OPTIONS=${HEAP_SIZE}
       - DB_HOST=postgresql
       - DB_USERNAME=${API_POSTGRES_USERNAME}
       - DB_PASSWORD=${API_POSTGRES_PASSWORD}
       - DB_DATABASE=${API_POSTGRES_DATABASE}
       - DB_SYNCHRONIZE=${API_DB_SYNCHRONIZE}
       - JWT_SECRET=${API_AUTH_JWT_SECRET}
       - GMAPS_API_KEY=${GMAPS_API_KEY}
+      - API_REDIS_SERVICE_PORT=${API_REDIS_SERVICE_PORT}
+      - API_REDIS_SERVICE_HOST=${API_REDIS_SERVICE_HOST}
     depends_on:
       - postgresql
+      - redis
+
+  redis:
```

Comment:
please review this. This is twice as complex as it needs to be. copy-pasting from the internet should be done with care ;)

Reply:
This is the best I could simplify. Let me know how this can be improved please :)